src/api-utils/getAuthUser.ts

```diff
@@ -17,13 +24,13 @@ export async function getAuthUser(this: CustomRequestThis, req: Request, _h: Res
   }
   const user = await findUserById(userSession._userId);
   if (!user) {
-    return Boom.notFound('User not found');
+    throw Boom.notFound('User not found');
   }
   const capabilityHeight = this.requiredCapabilities.shift() ?? ZERO;
 
   // This works for now while we design and code the capabilities system
   if ((user._roleId ?? ZERO) < capabilityHeight) {
-    return Boom.unauthorized('Only authorized users can access here');
+    throw Boom.unauthorized('Only authorized users can access here');
```

Comment:
it looks like this should be throw rather than return. Let me know if I should revert this...

Reply:
It's ok!